docs/DeveloperGuide.adoc

```diff
@@ -767,16 +834,169 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+=== Use case: 01 Bookmark a place
```

Comment:
You might consider omitting the numbering
Also consider using [discrete] for formatting.